ibis/backends/base/sql/alchemy/client.py

```diff
@@ -229,6 +230,43 @@ def truncate_table(
         t = self._get_sqla_table(table_name, schema=database)
         t.delete().execute()
 
+    def schema(self, name):
+        """Get a schema object from the current database for the schema named `name`.
+
+        Parameters
+        ----------
+        name : str
+
+        Returns
+        -------
+        schema : ibis Schema
+            The schema of the object `name`.
+
+        """
+        return self.database().schema(name)
+
+    @property
+    def current_database(self):
+        """The name of the current database this client is connected to."""
+        return self.database_name
+
+    def set_database(self, name):
+        raise NotImplementedError(
+            f'Cannot set database with {self.__class__.__name__} client. '
+            f'To use a different database, use `client.database("{name}")`'
+        )
+
+    def list_schemas(self):
+        warnings.warn(
+            '`list_schemas` is deprecated, use `list_databases` instead',
```

Comment:
can you add a test for this. Also we should create an issue with deprecations as a reminder to remove them.